evolution-chain/pallets/living-assets-evolution/src/lib.rs

```diff
@@ -217,8 +216,14 @@ impl<T: Config> Pallet<T> {
 	///
 	/// Every slot is identified by a unique `token_id` where `token_id = concat(slot #,
 	/// owner_address)`
-	fn slot_and_owner_to_token_id(slot_and_owner: (Slot, AccountIdOf<T>)) -> TokenId {
-		let (slot, owner) = slot_and_owner;
+	fn slot_and_owner_to_token_id(
+		slot: Slot,
+		owner: SlotOwnerId,
+	) -> Result<TokenId, pallet::Error<T>> {
```

Comment:
one more minor thing
```suggestion
	) -> Result<TokenId, Error<T>> {
```